MAUI/DataGrid/paging.md

```diff
@@ -297,9 +295,249 @@ public partial class MainPage : ContentPage
+
+{% tabs %}
+{% highlight c# %}
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        dataPager.PageChanging += DataPager_PageChanging;
+    }
+
+    private void DataPager_PageChanging(object sender, Syncfusion.Maui.DataGrid.DataPager.PageChangingEventArgs e)
+    {
+        // you can get the old page index and new page index here.
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
 ## DataPagerStyle
 
-The data grid allows you to change its appearance by creating a style class that overrides the `DataPagerStyle` and then assigning it to the `SfDataPager.DefaultStyle` property.
+The data pager allows you to change its appearance by changing the properties of `DataPagerStyle` and then assigning it to the `SfDataPager.DefaultStyle` property.
+
+The `SfDataPager` allows customizing the appearance using the following properties.
+
+<table>
+<tr>
+<th> Property </th>
+<th> Description </th>
+</tr>
+<tr>
+<td> {{'`DataPagerBackgroundColor`'| markdownify }} </td>
+<td> Gets or sets the background color of the SfDataPager.</td>
+</tr>
+<tr>
+<td> {{'`NavigationButtonBackgroundColor`'| markdownify }} </td>
+<td> Gets or sets the background color of the navigation button.</td>
+</tr>
+<tr>
+<td> {{'`NavigationButtonDisableBackgroundColor`'| markdownify }} </td>
+<td> Gets or sets the background color of the navigation button when it is in a disabled state.</td>
+</tr>
+<tr>
+<td> {{'`NavigationButtonDisableIconColor`'| markdownify }} </td>
+<td> Gets or sets the icon color of the navigation button when it is in a disabled state.</td>
+</tr>
+<tr>
+<td> {{'`NavigationButtonIconColor`'| markdownify }} </td>
+<td> Gets or sets the icon color of the navigation buttons.</td>
+</tr>
+<tr>
+<td> {{'`NumericButtonBackgroundColor`'| markdownify }} </td>
+<td> Gets or sets the background color for numeric button.</td>
+</tr>
+<tr>
+<td> {{'`NumericButtonSelectionBackgroundColor`'| markdownify }} </td>
+<td> Gets or sets background color of the selection numeric button.</td>
+</tr>
+<tr>
+<td> {{'`NumericButtonSelectionTextColor`'| markdownify }} </td>
+<td> Gets or sets the foreground color of the selection numeric button.</td>
+</tr>
+<tr>
+<td> {{'`NumericButtonTextColor`'| markdownify }} </td>
+<td> Gets or sets the foreground color of the numeric button.</td>
+</tr>
+<tr>
+<td> {{'`FirstPageButtonTemplate`'| markdownify }} </td>
+<td> Gets or sets the template of the first page button template.</td>
+</tr>
+<tr>
+<td> {{'`LastPageButtonTemplate`'| markdownify }} </td>
+<td> Gets or sets the template of the last page button template.</td>
+</tr>
+<tr>
+<td> {{'`NextPageButtonTemplate`'| markdownify }} </td>
+<td> Gets or sets the template of the next page button template.</td>
+</tr>
+<tr>
+<td> {{'`PreviousPageButtonTemplate`'| markdownify }} </td>
+<td> Gets or sets the template of the previous page button template.</td>

```

Comment:
Gets or sets the template for the previous page navigation button.

Reply:
Addressed the concerns provided.